docs/src/main/asciidoc/trace.adoc

```diff
@@ -1,4 +1,4 @@
-== Spring Cloud Sleuth
+== Stackdriver Trace
```

Comment:
Technically Cloud Trace...

Reply:
I know, I should probably use this opportunity to rename it across the board. We just can't rename class names and module names.

Reply:
It's actually a substantial change that needs to be done carefully.
Let's do it separately,
https://github.com/spring-cloud/spring-cloud-gcp/issues/2296